src/service/internal_tables_access.h

```diff
@@ -426,9 +427,7 @@ namespace ccf
         }
 
         const auto root = previous_service_last_signed_root->get().value();
```

Comment:
Is that not already a vector of bytes (optional)? If so, it seems like we could skip the assignment in the temporary?

Reply:
Made it a ref

Reply:
ASAN failed. Turns out `previous_service_last_signed_root->get()` is a temporary, and so gets assigned to `const auto root` as `T&&`, therefore move construction happens.

I don't think there's a need to sacrifice readability to avoid one extra move, but you may convince me.

Reply:
Talked offline, we have to copy from std array to std vector, that's why it's here.